### PR TITLE
Updates on previous entry with current changes

### PR DESCRIPTION
**Pull request by shyam-patel-kira**
This PR is a step towards closing #93 

- Merged the current updated content with few addition and curation.
- Made it more readable and organised the structure

### CODE REVIEW

File: docs/wiki/research/PBS/mev.md [hunk 1]
```diff
@@ -1,4 +1,19 @@
...
+
+## Evolution of MEV
+
+Maximal Extractable Value (MEV) originated during the proof-of-work Era, where it was known as "Miner Extractable Value." This terminology reflected the miners' ability to influence transaction processes, including their inclusion, exclusion, and sequencing. Following Ethereum's transition to proof-of-stake with The Merge, validators have taken over these critical functions, rendering mining obsolete within the protocol. Despite these changes, the mechanisms for value extraction remain in place, leading to the adoption of the term "Maximal Extractable Value" to address these activities.
+
+Although MEV has been around since the inception of Ethereum, it gained significant attention with the Rise of DeFi and Introduction of Flashloans. In the early days, MEV opportunities were primarily seized by outbidding rivals in the public mempool, marking the era known as Priority Gas Auction or PGA. Details about this chaotic era is captured in [Flashboys 2.0](https://arxiv.org/abs/1904.05234).
```
[the remark this answers is not in the document]
Comment (shyam-patel-kira) [reply docs/wiki/research/PBS/mev.md]: Updated this

File: docs/wiki/research/PBS/current-state.md [hunk 2]
```diff
@@ -31,12 +33,39 @@ Proposer-Builder Separation (PBS) also introduces a more intricate relationship
...
+
+### Undermined Censorship Resistance
 
-- Security: Introducing new actors and dependencies can create new attack vectors and vulnerabilities.
-- Centralization: If only a few powerful builders or relays dominate the ecosystem, it could lead to centralization and censorship concerns.
-- Coordination: Effective communication and coordination between builders, relays, and proposers are crucial for the smooth functioning of PBS.
+Another issue builder centralization might bring is putting at risk Ethereum's censorship resistance and integrity, as these dominant builders could, in theory, collude or be coerced into manipulating transaction flows or excluding specific transactions from being included in blocks, undermining the open and permissionless nature of the Ethereum network.
```
[the remark this answers is not in the document]
Comment (shyam-patel-kira) [reply docs/wiki/research/PBS/current-state.md:69]: Added some summary and linked an article for more info

File: docs/wiki/research/PBS/current-state.md [hunk 3]
```diff
@@ -31,12 +33,39 @@ Proposer-Builder Separation (PBS) also introduces a more intricate relationship
...
+
+As opposed to vanilla block creation in which the validator builds the block and adds it to the chain, PBS brings to the table many possible points of failure, such as the relays or escrows, which could potentially act maliciously or fail in their mission to provide data availability, disrupting the block proposal process, favoring certain builders, etc. Also, the commit-reveal scheme in escrows, if exploited, could give rise to MEV opportunities being stolen from the rightful block builders, undermining their performance.
+
+Of course, validators are still in charge of verifying the data in the blocks to make sure that builders adhere to the rigorous standards that make the Ethereum protocol robust. Thus, the aforementioned concerns are not likely to cause critical issues like the network halting. Nonetheless, the potential for these possible vulnerabilities to impact the fairness and security of block production suggests that they should not be overlooked and further research should be done to have them prevented.
```
[the remark this answers is not in the document]
Comment (shyam-patel-kira) [reply docs/wiki/research/PBS/current-state.md:65]: noted

File: docs/wiki/research/PBS/research.md [hunk 4]
```diff
@@ -1,3 +1,50 @@
...
+</figure>
+
+## Enshrined Proposer-Builder Separation (ePBS)
+
+Enshrined PBS involves embedding PBS mechanisms directly into Ethereum's consensus layer, which would have two main potential advantages:
+
+- Reduction in centralization risks: Moving PBS into the protocol layer could potentially reduce reliance on third parties with a tendency for centralization, aligning with Ethereum’s core values of decentralization and censorship resistance.
+- Security and stability: External dependencies and out-of-protocol software, such as relays, have shown vulnerabilities (e.g., "Low-Carb Crusader" attack). Integrating PBS into the Ethereum protocol can mitigate these risks and reduce the coordination costs associated with maintaining compatibility between various components.
+
+According to the article, ePBS, particularly through Two-Block HeadLock (TBHL) and optimistic relaying, presents a pathway towards addressing current challenges and enhancing the efficiency, security, and decentralization of block production and MEV extraction processes.
```
[the remark this answers is not in the document]
Comment (shyam-patel-kira) [reply docs/wiki/research/PBS/research.md]: It was referring to the article linked above...I agree it's confusing fixed it

File: docs/wiki/research/PBS/pbs.md [hunk 5]
```diff
@@ -1,193 +1,25 @@
-# Proposer-Builder Separation
+<!-- @format -->
```
[the remark this answers is not in the document]
Comment (shyam-patel-kira) [reply docs/wiki/research/PBS/pbs.md:1]: I've read it and rewrote the overlapping part. I've made sure I haven't deleted content...most of the seen deleted content from this particular file is moved/organised in a different suitable section, making it more readable

File: docs/wiki/research/PBS/current-state.md [hunk 6]
```diff
@@ -31,12 +33,39 @@ Proposer-Builder Separation (PBS) also introduces a more intricate relationship
...
+- Censorship resistance: Relays _can_ censor blocks and, being centralized, can be coerced by regulators to do so. This happened, for instance, when they were pressured to censor transactions interacting with addresses on the [OFAC sanction list](https://home.treasury.gov/news/press-releases/jy0916).
+- Trustlessness: Validators trust relays to provide a valid block header and to publish the full block once signed; builders trust relays not to steal MEV. Although betrayal of either would be detectable, dishonesty can be profitable even through a one-time attack.
+
+### Third party dependency
+
+On a similar note, the fact that PBS entails outsourcing the building of the blocks to entities that do not directly participate in Ethereum consensus could potentially lead to unexpected or unwanted consequences stemming from relying on third parties, such as trust issues, operational dependency and the introduction of single points of failure. Particularly the fact that the use of MEV-Boost is so widespread could be viewed as a dangerous third party dependency, since such a huge portion of Ethereum's new blocks are created using Flashbot's software.
```
[the remark this answers is not in the document]
Comment (shyam-patel-kira) [reply docs/wiki/research/PBS/current-state.md:57]: Mentioned this in the last commit and linked to the gist as well

**Review by taxmeifyoucan — APPROVED** [review]
Thanks for coordinating with others on this, looks good now

**Pull request merged**